package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "azure-sb",
+  "version": "0.11.0",
+  "description": "Azure ServiceBus SDK for Node.js",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "main": "./dist/lib/index.js",
+  "types": "./typings/lib/index.d.ts",
+  "dependencies": {
+    "async-lock": "^1.1.2",
+    "debug": "^3.1.0",
+    "is-buffer": "2.0.2",
+    "ms-rest": "^2.3.3",
+    "ms-rest-azure": "^2.5.5",
+    "rhea": "^0.2.14",
+    "tslib": "^1.9.2",
+    "uuid": "^3.2.1"
+  },
+  "devDependencies": {
+    "@types/uuid": "^3.4.3",
+    "@types/async-lock": "^1.1.0",
+    "@types/debug": "^0.0.30",
+    "@types/node": "^8.0.37",
```

Comment:
same comment as the CI. if we want to support node 6 then use the node 6 typings so you don't accidentally use something that isn't actually available

Reply:
I was thinking to restrict the min version of node.js to 8.x. What do you think?

Reply:
that's fine by me. if people request node 6 we can always refactor later